### ISSUE TITLE
Write README

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/beprof#2]

**Comment by matkrak** [comment]
What else should I put in readme file?
I could describe functionality, but for now its not really impressing. Once bigger part of beprof is ready I will make changes in README.md file.

**Comment by grzanka** [comment]
- 2-3 sentences about the purpose of the project
- installation (and deinstallation procedure) using pip+git (I prefer copy-pasteable instructions)
- some example usage - cuople of lines of the code which:
  - import beprof
  - get some profile data from the web
  - create profile object
  - calculate FWHM or other features
  - plot profile using matplotlib

**Comment by matkrak** [comment]
Already done. Take a look and give me feedback: is it OK or something is missing/needs to be changed?
